lib/v2/bulianglincom/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'bulianglin.com': {
+        _name: '不良林',
+        '.': [
+            {
+                title: 'blog',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/09f1a8d3814f4d563ebedb7632bebe50b354c13d/website/docs/routes/blog.mdx#L281
```suggestion
                title: '博客',
```

Reply:
已经修改